content/agent/how-to/run-agent-container.md

```diff
@@ -0,0 +1,98 @@
+---
+title: "Running the NGINX Agent in a Container"
+weight: 300
+docs: DOCS-000
+---
+
+## Overview
+
+This guide serves as a step-by-step guide to running the NGINX Agent in a container. It covers the basic setup needed to get the NGINX Agent up and running efficiently and securely.
+
+## Before you begin
+
+Before you begin this guide ensure:
+
+> **Note**: This guide uses docker but the Agent will also work with other application containers.
+
+- **Installed Docker:** Ensure Docker is installed and configured on your system. [You can download Docker here](https://www.docker.com/products/docker-desktop/).
+- **NGINX Agent Image:** You need access to the Docker image for the NGINX Agent. Find the appropriate image in your organization's registry or on Docker Hub if publicly available.
+- **NGINX Configuration File:** Prepare and validate your NGINX configuration files that the Agent will monitor.
+- **Credentials:** Acquire any necessary authentication tokens or credentials required for the NGINX Agent.
+
+1. Pull the NGINX Agent Container Image
+
+The NGINX Agent container image must be downloaded from a trusted source such as Docker Hub or a private container registry.
+
+Run the following command to pull the official image:
+
+```bash
+<!-- Registry HERE -->
+docker pull <Registry HERE>:latest
+```
+
+Ensure you are using the correct image version. Replace `latest` with the desired version tag if necessary.
+
+
+2. Create a Configuration File
```

Comment:
```suggestion
2. Create a configuration file
```
(In general) Only capitalize the 1st word of the sentence, proper nouns, product names, and API objects